docs/quickstart_compare.md

```diff
@@ -42,14 +44,13 @@ order_builder = rapi.create_compare_order("Simple Compare")
 order_builder = order_builder.criteria("Which logo looks better?")
 ```
 
-3. Add the paths to the images you want to classify:
+3. Add the paths to the images you want to classify (order will be randomized):
```

Comment:
I would add an extra comment about how you have to specify the pairs as a list of length 2. I.e., in the end you supply a list of pairs (list of lists). And then the clarification is that the order withing the pair will be randomized when displayed to the user